wiki/markdown/subscription-handling.md

```diff
@@ -6,26 +6,70 @@ become visible, which is populated by the information about the stored (if any)
 subscriptions in the connected database. This view is also the default view of
 the Eiffel Intelligence front-end, once the authentication (if enabled) is done.
 
-<img src="images/subscription_overview.png">
-</img>
+<kbd>
+    <img style="border:1px solid black" src="images/subscription_overview.png"></img>
+</kbd>
 
 #### _Add Subscription_
 This button opens a form with a number of fields to create a subscription
 through subscription endpoint (POST /subscriptions).
 #### _Upload Subscriptions_
-This button help uploads a subscription by opening a file explorer
+This button help uploads a subscription by opening a file explorer.
 #### _Bulk Delete_
 This button deletes all the selected subscriptions in the table from the
 database.
+#### _Bulk Download_
+This button downloads all the selected subscriptions in the table.
 #### _Reload_
 This button reloads the data from the database and refresh the data in the
-subscription table
+subscription table.
 #### _Get Template_
-This button downloads a subscription template
+This button downloads a subscription template.
 #### _EI Back-end Status_
 This button indicates, through its color, whether a back-end instance is
 connected with front-end or not. The green color means back-end is connected
 while red means no instance is connected.
 
+## Single Subscription Handling
+
+#### _Checkbox_
+Used to select one or many subscription for bulk usages. The top one is a
+select all checkbox.
+#### _SubscriptionName_
+The subsctiption name.
+#### _Date_
+The date when the subscription was created and saved in Eiffel Intelligence.
+#### _NotificationType_
+What kind of notification type the subscription uses.
+#### _NotificationMeta_
+The URL the subscription triggers or email adresses that will be notified on
```

Comment:
The URL which the subscription triggers. Or email addresses ...

Reply:
Done